changelog: Update changelog with Bluetooth RPC changes (#6085)
nrfconnect/sdk-nrf: doc/nrf/releases/release-notes-changelog.rst

* Updated some samples with support for :ref:`zephyr:thingy53_nrf5340` in non-secure configuration.
* :ref:`ble_llpm` sample - Added role selection.
  The user now selects the role for each board by typing "m" or "s" in the terminal emulator.

> * Changed the default sample for the :ref:`ble_rpc` to the :ref:`peripheral_uart` sample. This sample runs out of the box with a serialized Bluetooth Low Energy Host.
> * Added the :ref:`ble_rpc_host:` child image configuration overlay for the :ref:`peripheral_hids_mouse` sample as an example how to configure application running a serialized Bluetooth Low Energy Host.
>   The :ref:`peripheral_hids_mouse` runs out the box with the :ref:`ble_rpc` library.

Reviewer: ```suggestion
* :ref:`peripheral_uart` sample is now the default sample for the :ref:`ble_rpc` library.
  The sample runs out of the box with a serialized Bluetooth Low Energy Host.
* :ref:`peripheral_hids_mouse` sample now comes with the :ref:`ble_rpc_host` child image configuration overlay.
  The overlay shows how to configure application running a serialized Bluetooth Low Energy Host.
  The :ref:`peripheral_hids_mouse` runs out the box with the :ref:`ble_rpc` library.
```

Reviewer 2: The overlay shows how to configure **an** application...

Author: Done

This section provides detailed lists of changes by :ref:`library <libraries>`.

Bluetooth libraries
-------------------

> Updated:
> 
> * Added support for the GATT Server API serialization in the :ref:`ble_rpc` library.
> * Changed the config which enables the :ref:`ble_rpc` library from the :kconfig:`CONFIG_BT_RPC` to the :kconfig:`CONFIG_BT_RPC_STACK`.

Reviewer: ```suggestion
* :ref:`ble_rpc` library:

  * Added support for the GATT Server API serialization.
  * Changed the configuration option that enables the library from the :kconfig:`CONFIG_BT_RPC` to the :kconfig:`CONFIG_BT_RPC_STACK`.
```

Author: Done
